src/openms/include/OpenMS/ANALYSIS/ID/AScore.h

```diff
@@ -80,29 +80,77 @@ class OPENMS_DLLAPI AScore
     */
     PeptideHit compute(PeptideHit & hit, RichPeakSpectrum &real_spectrum, double fragment_mass_tolerance, bool fragment_mass_unit_ppm) const;
 
+  protected:
+  
+    template <class InputIterator1, class InputIterator2, class OutputIterator>
+    OutputIterator getSpectrumDifference_(InputIterator1 first1, InputIterator1 last1,
+      InputIterator2 first2, InputIterator2 last2, OutputIterator result) const
+    {
+      while (first1!=last1 && first2!=last2)
+      { 
+        if ((roundf(first1->getMZ() * 100) / 100) < (roundf(first2->getMZ() * 100) / 100)) 
```

Comment:
couold you add a textual description what this function does?

/ 100 is not needed and you could use round